### ISSUE TITLE
sticky headers

### ISSUE THREAD

**Issue by petersnoopy** [h6ah4i/android-advancedrecyclerview#1]
A nice feature would be to able to integrate section headers in your recyclerview.

**Comment by h6ah4i** [comment]
Hi. Thanks for your suggestion.

I thinks this [timehop/sticky-headers-recyclerview](https://github.com/timehop/sticky-headers-recyclerview) library will work with this library. I'll test them later.

**Comment by jpshelley** [comment]
@petersnoopy I would agree with @h6ah4i timehops lib works well, and its modular enough that you could use both in conjunction

**Comment by petersnoopy** [comment]
Thanks, i will try your forked project...

**Comment by guidedways** [comment]
This would be awesome to have! It would be even great if the 'Grouped' sections could in fact be sticky (the ones enclosing child items).

**Comment by silviahisham** [comment]
+1

**Comment by csbenz** [comment]
+1

**Comment by pytel87** [comment]
+1

**Comment by caiyoufei** [comment]
+1
